Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -1051,6 +1051,10 @@ theorem StructureGroupoid.mem_maximalAtlas_of_mem_groupoid {f : PartialHomeomorp
   rintro e (rfl : e = PartialHomeomorph.refl H)
   exact ⟨G.trans (G.symm hf) G.id_mem, G.trans (G.symm G.id_mem) hf⟩
 
+theorem StructureGroupoid.maximalAtlas_mono {G G' : StructureGroupoid H} (h : G ≤ G') :
```

Comment:
Happy to approve this "immediately" if you extract it into a separate PR. (If you think the book-keeping is worth it.)
The same applies to other such clean-ups, I believe I saw a few more lines somewhere in this PR.

Reply:
Good idea. I've put in #19807 all the material of this PR I could find that makes sense as a standalone PR.